{{cookiecutter.project_slug}}/_/deployment/python/deployment/application/base/cronjob.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: cron-backup
+spec:
+  schedule: "1 1 * * *"
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 1
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: application
+            image: IMAGE
+{%- if cookiecutter.framework == 'Django' %}
+            command: ['/bin/bash', '-c']
+            args: 
+              - python manage.py dumpdata -o backups/db.json ;
+              - cat backups/db.json
+            envFrom:
+            - secretRef:
+                name: application
+{%- else %}
+            command: ['echo', 'hello world']
```

Comment:
That may fail (see other comments).

Reply:
I think it's ok for it to give error messages to prompt the user to replace it with a real job.